results/2020/overview.mdx

```diff
@@ -5,13 +5,26 @@ position: 1
 
 # 1.Overview
 
-overview about the survey
-
-- the is the first version and we are planing to make every year.
-- lunch during blablaConf and 1 month to collect submission
-- privacy-first
-- 49 Question and 4 categories
-- project is open source
-- Thanks to contributors and everyone who shared the survey
-- 2287 submission
--
+We, at [GeeksBlaBla](https://geeksblabla.com/) are a community of passionate IT professionals, sharing latest and hottest tech in darija, the moroccan dialect.
```

Comment:
hmm Not sure but I would prefer DevC Morocco instead of GeeksBlabla, @aboullaite WYT? 
https://www.facebook.com/groups/DevC.Casablanca

Reply:
Yep! DevC morocco community.